server/src/test-utils/environments/mikro-environment.ts

```diff
@@ -11,10 +11,11 @@ export class MikroTestEnvironment extends TestEnvironment<MikroEnvironmentVariab
   protected waterparkExpress!: http.WaterparkExpress
 
   public async setup(): Promise<MikroEnvironmentVariables> {
-    // must set environment variable DATABASE_URL to postgresql://loolabs:loolabs@localhost/clubs
-    // to access postgres container on Docker
-    // TODO: fix integration testing
     this.mikroDB = await db.setupMikroDB({ debug: false })
+
+    const migrator = await this.mikroDB.orm.getMigrator()
+    if (process.env.IS_CI === 'true') await migrator.up()
```

Comment:
The migrator is already run with `docker compose up`

Reply:
where does that happen?

Reply:
`MikroORM.init()` I believe

Reply:
discussed this in call: `.init` appears to be run during local integration tests because we spin up the server and then switch it over to run the integration tests

there is another way where we just directly run the integration tests, in which `.init` won't occur

thus I think we should run it regardless of `IS_CI`

```suggestion
    const migrator = await this.mikroDB.orm.getMigrator()
    await migrator.up()
```